zsh completions are broken

Please don't install bash completion scripts to zsh. They don't work. Zsh itself already ships a `_adb` completion script which works great.

Broken by #23.

So I just made a simple patch for it.

Unfortunately I cannot test the build as master is in a weird state for me now:

```
Applying: libsepol: fix inclusion of dso.h header file
-- Applying patches for: e2fsprogs
Applying: linux/types.h seem to already define __bitwise
-- Applying patches for: incremental_delivery
fatal: previous rebase directory .git/rebase-apply still exists but mbox given.
CMake Error at vendor/CMakeLists.txt:30 (message):
  Couldn't apply patches for incremental_delivery
```

--- a/completions/CMakeLists.txt
+++ b/completions/CMakeLists.txt
@@ -1,2 +1,1 @@
 add_subdirectory(bash)
-add_subdirectory(zsh)

--- a/completions/zsh/CMakeLists.txt
+++ b/completions/zsh/CMakeLists.txt
@@ -1,9 +1,0 @@
-set(ZSH_COMPLETION_DIR "${CMAKE_INSTALL_FULL_DATADIR}/zsh/site-functions")
-
-configure_file(adb.in _adb)
-configure_file(fastboot.in _fastboot)
-
-install(FILES
-	"${CMAKE_CURRENT_BINARY_DIR}/_adb"
-	"${CMAKE_CURRENT_BINARY_DIR}/_fastboot"
-	DESTINATION "${ZSH_COMPLETION_DIR}")

--- a/completions/zsh/adb.in
+++ b/completions/zsh/adb.in
@@ -1,8 +1,0 @@
-# Bash completions for adb.
-# See https://github.com/nmeum/android-tools/issues/22
-
-function check_type() {
-	type "$1"
-}
-
-source "@COMPLETION_COMMON_DIR@/adb"

--- a/completions/zsh/fastboot.in
+++ b/completions/zsh/fastboot.in
@@ -1,8 +1,0 @@
-# Bash completions for fastboot.
-# See https://github.com/nmeum/android-tools/issues/22
-
-function check_type() {
-	type "$1"
-}
-
-source "@COMPLETION_COMMON_DIR@/fastboot"

